pkg/apis/minio.min.io/v1/helper.go

```diff
@@ -53,14 +53,15 @@ import (
 const (
 	WebhookAPIVersion       = "/webhook/v1"
 	WebhookDefaultPort      = "4222"
-	WebhookMinIOArgsSecret  = "minio-args-secret"
+	WebhookSecret           = "minio-args-secret"
```

Comment:
```suggestion
	WebhookSecret           = "operator-webhook-secret"
```

Reply:
This may make it backward-incompatible @nitisht ??

Reply:
No, it won't. Operator will simply create the secret with new name and start using that.